go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/spf13/afero v1.14.0
 	github.com/spf13/cobra v1.9.1
 	github.com/spf13/pflag v1.0.6
+	golang.org/x/mod v0.25.0
```

Comment:
Not sure why this is changing in this PR. Was the `go.mod` file not tidy on master?

Reply:
Go changes, misisng update done in another PR